auth/oidc_verify.go

```diff
@@ -58,6 +90,13 @@ func VerifyClaims(rawIDToken, clientID, issuer string) (*Identity, error) {
 		Email:    identityJson.Email,
 	}
 
+	if usernameClaim != "" {
+		var ok bool
+		if identity.Username, ok, err = claimString(payload, usernameClaim); !ok {
```

Comment:
Instead of doing a second unmarshal to pull claims out of the payload, why not include claims in IdentityJSON in the first place?  I wouldn't think the overhead of unmarshalling claims (even if usernameClaim isn't defined) would be particularly significant?

Reply:
If you went a step further and also included Claims in Identity instead of Username, then you could avoid passing usernameClaim around everywhere, and better encapsulate the usernameClaim handling in GetOIDCUsername.  I think that does a better job of separating the functionality, and leaves Identity as just a straight representation of the token.

Reply:
Sure Adam, added changes to include claims in both Identity and IdentityJson; individual properties unmasrshalled ito IdentityJson are removed from the claims.